frontend/frontend/src/components/TheNavBar/components/AccountDropdownContents.vue

```diff
@@ -28,18 +27,20 @@
         ></b-form-input>
         <!--Feedback for if input is invalid (in the false state)-->
         <b-form-invalid-feedback id="feedback-invalid-signin-passwd">
-            Passwords are at least 8 characters
-          </b-form-invalid-feedback>
+          Passwords are at least 8 characters
```

Comment:
```suggestion
          Please enter a password with at least 8 characters
```

Sounds more like an error message?